tests/pytorch/r2.2/sd-model.libsonnet

```diff
@@ -62,11 +62,10 @@ local utils = import 'templates/utils.libsonnet';
         # mv quantize.py ~/.local/lib/python3.8/site-packages/taming/modules/vqvae/
 
         # taming-transformers and CLIP override existing torch and torchvision so we need to reinstall
-        # TODO change back to torch2.1 once pytorch released torch2.1
         pip uninstall -y torch torchvision
-        pip install --user --pre --no-deps torch torchvision --extra-index-url https://download.pytorch.org/whl/test/cpu
-        pip install https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.1.0-cp310-cp310-manylinux_2_28_x86_64.whl
-        pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html
+        pip3 install --user --pre torch torchvision --index-url https://download.pytorch.org/whl/test/cpu
```

Comment:
let's use `pip3 install torch==2.2.0 torchvision torchaudio --index-url https://download.pytorch.org/whl/test/cpu` according to [blog](https://dev-discuss.pytorch.org/t/pytorch-2-2-rc1-is-produced-for-pytorch-audio-vision-and-text/1717)